lib/fd-manager.js

```diff
@@ -1,7 +1,5 @@
 (function(){
-
-   "use strict";
```

Comment:
I read at some node.js site that the convention was not to use 'use strict', having double-checked I found out I was wrong. So I added the use strict again. Sorry..